### ISSUE TITLE
Broken with pg 7

### ISSUE THREAD

**Issue by jake-lambert-volusion** [sequelize/sequelize-auto#397]
Hello - Really useful module here - thanks for your great work!

I cannot generate models with pg@^7 which no longer supports query.on(). 

For reference, see [pg upgrade notes on this topic](https://node-postgres.com/guides/upgrading#client.query(...).on).

Would you be willing to update for compatibility (I'm open to trying to offer a PR, but I've never looked at your code so don't really know what that's going to entail)? 

Jake

```
jake.lambert@M02961volusion-api-v2 (feature/V2E-1508-products-rest *$%<)$ npx sequelize-auto -h localhost -d <REDACTED> -u <REDACTED> -x <REDACTED> -s <REDACTED> -e postgres -z -o testmodels
TypeError: connection.query(...).on is not a function
    at /Users/jake.lambert/Documents/repos/volusion-api-v2/node_modules/sequelize-auto/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:179:31
    at Promise._execute (/Users/jake.lambert/Documents/repos/volusion-api-v2/node_modules/bluebird/js/release/debuggability.js:303:9)
    at Promise._resolveFromExecutor (/Users/jake.lambert/Documents/repos/volusion-api-v2/node_modules/bluebird/js/release/promise.js:483:18)
    at new Promise (/Users/jake.lambert/Documents/repos/volusion-api-v2/node_modules/bluebird/js/release/promise.js:79:10)
    at /Users/jake.lambert/Documents/repos/volusion-api-v2/node_modules/sequelize-auto/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:178:12
    at PassThroughHandlerContext.finallyHandler (/Users/jake.lambert/Documents/repos/volusion-api-v2/node_modules/bluebird/js/release/finally.js:57:23)
    at PassThroughHandlerContext.tryCatcher (/Users/jake.lambert/Documents/repos/volusion-api-v2/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/jake.lambert/Documents/repos/volusion-api-v2/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/Users/jake.lambert/Documents/repos/volusion-api-v2/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/Users/jake.lambert/Documents/repos/volusion-api-v2/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/Users/jake.lambert/Documents/repos/volusion-api-v2/node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (/Users/jake.lambert/Documents/repos/volusion-api-v2/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/Users/jake.lambert/Documents/repos/volusion-api-v2/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/Users/jake.lambert/Documents/repos/volusion-api-v2/node_modules/bluebird/js/release/async.js:17:14)
    at processImmediate (internal/timers.js:439:21)
Done!
```

**Comment by yedhink** [comment]
Same problem. Tried downgrading to `pg@6` as mentioned in #184 , but it results in empty output after 1 second of running.

**Comment by steveschmitt** [comment]
Fixed as of `sequelize-auto@0.5.0`
